### PR TITLE
Fix hardcoded pickup

### PR DESCRIPTION
**Pull request by bramvankooten**

### CODE REVIEW

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-pickup.html [hunk 1]
```diff
@@ -24,32 +34,45 @@
...
+            data: Object,
+            endpoint: String,
+          };
+        }
 
-        onResponse: function(e, request) {
-          if (request.succeeded) {
+        onResponse(e, request) {
+            if (!request.succeeded) {
+                this.$.getPickupError.setError("We cannot find a TicketOption with that name. Try again");
```
Comment (Fastjur) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-pickup.html:51]: Is a failure always indicative of "cannot find a TicketOption with that name" ?

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-pickup.html [hunk 2]
```diff
@@ -24,32 +34,45 @@
...
                 "Ticket ID": data.id,
                 "Name": `${data.owner.profile.firstName} ${data.owner.profile.lastName}`,
                 "Email": data.owner.email,
                 "Address": `${data.owner.profile.address} ${data.owner.profile.zipcode} ${data.owner.profile.city}`,
                 "Phone number": data.owner.profile.phoneNumber,
             };
-        },
+        }
+
+        getPickup() {
```
Comment (Fastjur) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-pickup.html:68]: Correct me if I'm wrong. But isn't the naming convention in the lancie repos 'handleTapGetPickup'? If not, then you can dismiss this.

**Review by Fastjur — APPROVED** [review]

**Comment by Fastjur** [comment]
I'm assuming this can be closed as the pickup service is no longer offered? @bramvankooten @ArdyZ

**Comment by ArdyZ** [comment]
De pick up service is indeed not offered any more, is it possible to archive it?

**Comment by Fastjur** [comment]
Sorry, what do you mean archive it. Do you mean close this PR? I think that would be best if indeed the pickup service is not offered anymore.

**Comment by ArdyZ** [comment]
Keep it, but not merge it. As this is this first year we are not doing it any more. If it turns out that from feedback we need to do it again next year then we still have this one.

**Comment by bramvankooten** [comment]
If the pr is closed, the branch is still there, so the changes will not be lost

**Comment by ArdyZ** [comment]
Perfect

**Pull request closed without merging**